dffml/source/csv.py

```diff
@@ -113,7 +114,18 @@ class CSVSource(FileSource, MemorySource):
             repo_data = {}
             # Parse headers we as the CSV source added
             csv_meta = {}
-            for header in self.CSV_HEADERS:
+            row_keys=[]
+            # getting all keys starting with "prediction","confidence"
+            for _header in self.CSV_HEADERS:
+                row_keys.extend(list(
+                     filter(
+                    lambda x: x.startswith(_header),
```

Comment:
I think we want `_header + "_"` here. Also, variables shouldn't be prefixed with `_` unless we aren't using them (reference: https://stackoverflow.com/a/5893946/12310488 bullet point 3)